### ISSUE TITLE
Don't suggest packages from jakub-onderka namespace

### ISSUE THREAD

**Issue by tomasfejfar** [php-parallel-lint/PHP-Parallel-Lint#19]
I checked that composer.json is correct (and it is) but composer suggested installing a package from old namespace today:

```
  - Installing php-parallel-lint/php-parallel-lint (v1.1.0): Cloning ec813bf3dc from cache
php-parallel-lint/php-parallel-lint suggests installing jakub-onderka/php-console-highlighter (Highlight syntax in code snippet)
```

**Comment by tomasfejfar** [comment]
Oh, it's because the fix is not yet released! Last released version is ec813bf3dc where the issue is not yet fixed https://github.com/php-parallel-lint/PHP-Parallel-Lint/blob/ec813bf3dc/composer.json#L16

**Comment by grogy** [comment]
Hello you have a right. New release [is comming](https://github.com/php-parallel-lint/PHP-Parallel-Lint/releases/tag/v1.2.0) :-)
